### PR TITLE
Updates needed for website publication

### PR DESCRIPTION
**Pull request by alaudazzi**
1. Added `[partintro]` to `accessing-services.asciidoc`
2. Added `accessing-services.asciidoc` to `index.asciidoc`

### CODE REVIEW

File: docs/index.asciidoc [hunk 1]
```diff
@@ -7,3 +7,4 @@
 include::{asciidoc-dir}/../../shared/versions70.asciidoc[]
 include::{asciidoc-dir}/../../shared/attributes.asciidoc[]
 include::k8s-quickstart.asciidoc[]
+include::accessing-services.asciidoc[]
```
Comment (pebrc) [inline docs/index.asciidoc]: Will this not include the new doc on the same page as the quickstart, given that we build with `--single`? Let me know if I got my asciidoc all wrong but is it not time to have two separate documents now and an index page linking to them?

Comment (alaudazzi) [reply docs/index.asciidoc]: We are going to have two separate docs and an index. Will share a preview asap.

**Comment by pebrc** [comment]
jenkins test this please

**Comment by thbkrkr** [comment]
Now that we have two separate doc pages, I find the index a bit empty. Shouldn't we add a little intro? Maybe start with the content of the [README](https://github.com/elastic/cloud-on-k8s/blob/master/README.md#elastic-cloud-on-kubernetes-eck)? This can be done in another PR.

**Pull request merged**